lib/between_meals/repo/hg.rb

```diff
@@ -115,9 +115,17 @@ def head_parents
       end
 
       def last_author
-        { :email => template('author').match(/^(.*)<(.*)>$/)[2] }
-      rescue
-        { :email => nil }
+        # rubocop:disable Lint/HandleExceptions
+        begin
+          return { :email => template('author').match(/^(.*)<(.*)>$/)[2] }
+        rescue
```

Comment:
That rule is about the rescue block being empty, which is fine here, not about catching specific exception classes